fcrepo-connector-file/src/main/java/org/fcrepo/connector/file/FedoraFileSystemConnector.java

```diff
@@ -324,12 +322,10 @@ protected void touchParent( final String id ) {
 
     /* Overriding so unit test can mock. */
     @Override
-    @VisibleForTesting
```

Comment:
Modeshape allows you to optionally store binary files in Cassandra, so it lists this as a dependency. In principle, modeshape's MANIFEST.MF file should specify the dependency as optional, but we are not so lucky. That is, the dependency is unnecessary from the perspective of fcrepo-kernel-modeshape, but from the modeshape-jcr module, it is a hard dependency.

The confusing thing, though, is that the datastax module shows the guava import can be anything in the version range [14-19) https://github.com/datastax/java-driver/blob/2.0/driver-core/pom.xml#L192, but when I actually try to start that bundle in Karaf, it claims to require only version 16.

And no, I can't prevent the cassandra drivers from needing to be loaded. Nor have I had any luck with setting the resolution of that bundle to optional. The problem appears to be that it is a transitive dependency, and so it's rather out of our control unless we start doing things like creating a fat jar, shading, etc, etc (which I'd prefer to avoid).